src/common/files_js.ml

```diff
@@ -13,10 +13,9 @@
 open Utils
 open Modes_js
 
-let flow_extensions = [
-    ".js"  ;      (* Standard JavaScript files *)
-    ".jsx" ;      (* JavaScript files with JSX *)
-  ]
+let get_flow_extensions root =
```

Comment:
Agreed, so we do not need a `get_options` function in `modules_js.ml` if we store `options` at `Modules_js`, right ?